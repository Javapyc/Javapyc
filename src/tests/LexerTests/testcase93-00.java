class
public static extends
void
int boolean if
else
while return null true
false
this new String main System.out.println