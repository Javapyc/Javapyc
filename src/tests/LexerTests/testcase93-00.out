ReservedWord, class
ReservedWord, public
ReservedWord, static
ReservedWord, extends
ReservedWord, void
ReservedWord, int
ReservedWord, boolean
ReservedWord, if
ReservedWord, else
ReservedWord, while
ReservedWord, return
ReservedWord, null
ReservedWord, true
ReservedWord, false
ReservedWord, this
ReservedWord, new
ReservedWord, String
ReservedWord, main
ReservedWord, System.out.println
